crypto/bls_no_relic.go

```diff
@@ -0,0 +1,124 @@
+//go:build !relic
+// +build !relic
```

Comment:
Couldn't we remove this since the minimum Go version is 1.16 where this is no longer supported?

Reply:
At least one of the 2 lines is required to mention that this file is only compiled when the build doesn't contain the "relic" tag. 
The extra line is added by my IDE, although I'm using Go 1.17.5.